cmd/join.go

```diff
@@ -84,11 +85,13 @@ func newJoinRunner() *runner {
 }
 
 func joinPhasesSetup(cmd *cobra.Command, args []string) (*phaseInput, error) {
-	endpoint := args[0]
-	if _, err := url.Parse(endpoint); err != nil {
-		return nil, fmt.Errorf("endpoint %q must be a valid URL: %s", endpoint, err)
+	endpoints := strings.Split(args[0], ",")
+	for _, endpoint := range endpoints {
+		if _, err := url.ParseRequestURI(endpoint); err != nil {
+			return nil, fmt.Errorf("endpoint %q must be a valid URL: %s", endpoint, err)
```

Comment:
Nit: I think we should use %w (`fmt.Errorf("...: %w", ..., err)`) for any errors now; at some stage we'll have to go back and fix the existing invocations, but we should probably start using the chaining form as it's a few go versions old at this point.

Not a blocker as we have tons of %v and %s already though!